File: Objective-C/FitBitSteps/Globals.m
//
//  Globals.m
//  FitBitSteps
//
//  Created by Daniel Hsu on 9/5/17.
//  Copyright © 2017 Mango. All rights reserved.
//

#import "Globals.h"

@implementation Globals

static NSString* _accessToken;
static NSString* _userID;

+ (NSString*) accessToken {
	return _accessToken;
}

+ (void) setAccessToken:(NSString *)accessToken {
	_accessToken = accessToken;
}

+ (NSString*) userID {
	return _userID;
}

+ (void) setUserID:(NSString *)userID {
	_userID = userID;
}

@end

File: FitBitSteps/Globals.swift
//
//  Globals.swift
//  FitBit Steps
//
//  Created by Daniel Hsu on 7/18/17.
//  Copyright © 2017 Mango. All rights reserved.
//

import Foundation

let authURL: String = "https://www.fitbit.com/oauth2/authorize"
let callbackURI: String = "mhStepTracker://oauth-callback"
let tokenURL: String = "https://api.fitbit.com/oauth2/token"
let clientSecret: String = "45d700afa2244b04fc30f26b205eeb8b"
let clientID: String = "228NRR"
var accessToken: String = ""

var stepsData: Dictionary<String, Any>? = nil
